tests/basic_fuzzer.cpp

```diff
@@ -633,24 +632,24 @@ TEST(basic_fuzz) {
             reinterpret_cast<char32_t *>(input.data()),
             input.size() / sizeof(char32_t), reinterpret_cast<char16_t *>(output.data()));
         if(is_ok_utf8.second ? (utf8_to_utf16.second == 0 || utf8_to_utf32.second == 0) : (utf8_to_utf16.second > 0 || utf8_to_utf32.second > 0)) {
-          std::cout << (is_ok_utf8.second ? "UTF-8 is ok" : "UTF-8 is not ok") << std::endl;
-          std::cout << " size = " << input.size() << std::endl;
-          std::cout << "  implementation.convert_utf8_to_utf16.second return " << utf8_to_utf16.second << std::endl;
-          std::cout << "  implementation.convert_utf8_to_utf32.second return " << utf8_to_utf32.second << std::endl;
+          printf("%s\n", (is_ok_utf8.second ? "UTF-8 is ok" : "UTF-8 is not ok"));
+          printf(" size = %llu\n", input.size());
+          printf("  implementation.convert_utf8_to_utf16.second return %llu\n", utf8_to_utf16.second);
+          printf("  implementation.convert_utf8_to_utf32.second return %lli\n", utf8_to_utf32.second);
         }
         ASSERT_TRUE(is_ok_utf8.second ? (utf8_to_utf16.second > 0 && utf8_to_utf32.second > 0) : (utf8_to_utf16.second == 0 && utf8_to_utf32.second == 0));
         if(is_ok_utf16.second ? (utf16_to_utf8.second == 0 || utf16_to_utf32.second == 0) : (utf16_to_utf8.second > 0 || utf16_to_utf32.second > 0)) {
-          std::cout << (is_ok_utf16.second ? "UTF-16 is ok" : "UTF-16 is not ok") << std::endl;
-          std::cout << " size = " << input.size() / sizeof(char16_t) << std::endl;
-          std::cout << "  implementation.convert_utf16_to_utf8.second return " << utf16_to_utf8.second << std::endl;
-          std::cout << "  implementation.convert_utf16_to_utf32.second return " << utf16_to_utf32.second << std::endl;
+          printf("%s\n", (is_ok_utf16.second ? "UTF-16 is ok" : "UTF-16 is not ok"));
+          printf(" size = %llu\n", input.size() / sizeof(char16_t));
+          printf("  implementation.convert_utf16_to_utf8.second return %llu\n", utf16_to_utf8.second);
```

Comment:
The recommended expression to print a `size_t` value is `%zu` and not `%llu`.

See https://en.cppreference.com/w/c/types/size_t